package.json

```diff
@@ -12,22 +12,21 @@
   },
   "dependencies": {
     "bossy": "3.x.x",
-    "code": "4.1.x",
-    "diff": "3.3.x",
-    "eslint": "4.7.x",
+    "code": "5.1.x",
```

Comment:
No no no. Time to remove it and put it back in dev deps. People should get their assertion module explicitly.